src/app.js

```diff
@@ -526,8 +526,14 @@ var run = function () {
     // populate version dropdown with all available compiler versions
     //
 
-    // NOTE: the list currently is oldest first
-    $.each(data.builds.reverse(), function (i, build) {
+    // Sort builds according to semver
+    var builds = data.builds.sort(function (a, b) {
+      a = a.version + '-' + a.prerelease;
+      b = b.version + '-' + b.prerelease;
+      return semver.lt(a, b);
```

Comment:
What about `semver.compare(a, b)`? I think this function is required to return -1 / 0 / 1 and not true / false.